pkg/resources/cloudconfig/cloudconfig.go

```diff
@@ -100,6 +100,14 @@ func CloudConfig(
 
 	case cloud.Openstack != nil:
 		manageSecurityGroups := dc.Spec.Openstack.ManageSecurityGroups
+		loadBalancerProvider := ""
+		loadBalancerMethod := "ROUND_ROBIN"
```

Comment:
Please create a constant for this at the package/file level to make it more visible.